### ISSUE TITLE
Feature Request Slide Menu positioning

### ISSUE THREAD

**Issue by xreider** [primefaces/primereact#2579]
Hello! Is there any methods to position Slide Menu relative to the triggering element  ?

**Comment by xreider** [comment]
I mean positions like 'center-bottom', 'right-top' and others like in [tippy.js library](https://atomiks.github.io/tippyjs/#placements)

**Comment by melloware** [comment]
oh you mean like "where" it slides to right now it just has collision detection and either slides up or down depending on where the button is on the screen.  You want control over where it renders.  Understood.

**Comment by xreider** [comment]
> oh you mean like "where" 

Yes. And if I set 'top', but the triggering element on the top edge of screen, menu would open of the bottom of triggering element. etc

While there is no such function, I am forced to use tippy, inside which I embed the primereact's slide menu or use react components with react-components-group

**Comment by melloware** [comment]
This does seem like a nice feature request.  I know Primereact tooltips have positioning and there is this ticket for adding more positions: https://github.com/primefaces/primereact/issues/2438

It may be they could reuse the same code for SlideMenu.
